.github/actions/setup-environment/action.yml

```diff
@@ -105,3 +106,10 @@ runs:
         ./eden setup -v debug --grub-options='set_global dom0_extra_args "$dom0_extra_args eve_install_zfs_with_raid_level "'
       shell: bash
       working-directory: "./eden"
+
+    - name: Start and Onboard
```

Comment:
Looks like you disable setup in workflows and move its logic here. Do you have any reason for that?

Reply:
Will we keep our collect-info and other debugging logic in case of failure on onboarding stage, which is quite predictable?

Reply:
Hey @giggsoff , nice seeing you here :) So I did it purely from backward compatibility. For some reason escript tests didn't want to onboard through test. 

Yes, we keep all collect-info logic there, no changes in that regard